### DISCUSSION TITLE
How to switch token for metamask?

### DISCUSSION THREAD

**Discussion by victorpavlenko** [wevm/wagmi#2284] [Q&A]
Like for balance, I can send token and got my **USDC** balance, but when I _usePrepareSendTransaction_ it still sending **BNB** and I have no idea how to switch, any help?

```
const balance = useBalance({
    address: address,
    token: currentCrypto.addr
});
```

**Comment by di-sukharev** [comment]
+1, WHY can't I specify token address to be sent?!
